src/DynamoCore/Scheduler/UpdateGraphAsyncTask.cs

```diff
@@ -25,7 +25,7 @@ public override TaskPriority Priority
             get { return TaskPriority.AboveNormal; }
         }
 
-        protected IEnumerable<NodeModel> ModifiedNodes { get; set; }
+        public IEnumerable<NodeModel> ModifiedNodes { get; set; }
```

Comment:
Why do we need to make this public? Can we keep the setter for ModifiedNode protected?